subiquity/models/welcome.py

```diff
@@ -23,7 +23,9 @@ class WelcomeModel(object):
     """ Model representing language selection
     """
 
-    supported_languages = ['English']
+    supported_languages = [
+        ('English', 'en_US'),
+        ]
     selected_language = None
```

Comment:
Is this intentional? in my in-progress i18n branch I change supported_languages to have a different structure, and it's not that.

Reply:
Oh I was just doodling, I'll change this back.